huaweicloud/services/dataarts/resource_huaweicloud_dataarts_architecture_code_table.go

```diff
@@ -0,0 +1,758 @@
+				Computed:    true,
+				ForceNew:    true,
+				Description: "The region where the user group is located.",
+			},
+			"workspace_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "The workspace ID.",
+			},
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The name of the code table.",
+			},
+			"code": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The code of the code table.",
+			},
+			"directory_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The directory ID of the code table.",
+			},
+			"fields": {
+				Type:     schema.TypeList,
+				Required: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"name": {
+							Type:        schema.TypeString,
+							Required:    true,
+							Description: "The name of a field.",
+						},
+						"code": {
+							Type:        schema.TypeString,
+							Required:    true,
+							Description: "The code of a field.",
+						},
+						"type": {
+							Type:        schema.TypeString,
+							Required:    true,
+							Description: "The type of a field.",
+						},
+						"values": {
+							Type:        schema.TypeList,
+							Optional:    true,
+							Elem:        &schema.Schema{Type: schema.TypeString},
+							Description: "The values of a field.",
+						},
+						"description": {
+							Type:        schema.TypeString,
+							Optional:    true,
+							Description: "The description of a field.",
+						},
+						"ordinal": {
+							Type:        schema.TypeInt,
+							Computed:    true,
+							Description: "The ordinal of a field.",
+						},
+						"table_id": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "The ID of the code table.",
+						},
+						"id": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "The ID of the field.",
+						},
+					},
+				},
+				Description: "The fields information of the code table.",
+			},
+			"description": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "The description of the code table.",
+			},
+			"directory_path": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The directory path of the code table.",
+			},
+			"created_by": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The user who created the code table.",
+			},
+			"created_at": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The time when the code table was created.",
+			},
+			"updated_at": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The time when the code table was updated.",
+			},
+			"status": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The status of the code table.",
+			},
+			"id": {
```

Comment:
id is a reserved field and does not need to be declared in scehma

Reply:
fixed